README.md

```diff
@@ -23,23 +23,29 @@ cargo install cargo-wipe
 cargo wipe --help
 ```
 
-## target
+## Usage
+
+To delete all folders called `folder-name` folders run

```

Comment:
This states `To delete all folders`, but by running the command below the user will only get a dry run. Is there a way to make this more obvious? The note saying that this will be a dry run further down feels a bit too further down.

Reply:
Good point. I'll update it.